docs/user_documentation/admin-features/guide-settings.md

```diff
@@ -1,10 +1,37 @@
-# Administrator features
+# Settings
+There are different items where you can configure settings in MOLGENIS:
+- [Application settings](#application-settings)
+- [Authentication settings](#authentication-settings)
+- [Mail settings](#mail-settings)
+- [DataExplorer settings](../finding-data/guide-explore.md)
+- OpenCPU settings
 
+## Application settings
+There are a three main sections in the application settings.
 
+- General settings
+- Tracking settings
+- [ReCaptcha settings](#recaptcha-settings)
 
+### ReCaptcha settings
+You can configure reCaptcha in MOLGENIS by setting up 2 services.
```

Comment:
I do not understand this line, could you elaborate on these 2 services

Reply:
maybe add short description of recaptcha proces and feature, with link to google recaptcha page